client/model/adminModel.js

```diff
@@ -32,5 +33,9 @@ define(['knockout', 'jquery', './loginModel'],
                 });
         };
 
+        admin.toggleProgress = function() {
```

Comment:
Initially it's going to be disabled until I can test how good the data is/how it affects performance/how people want it represented (sec, loading bar, etc). If al that goes well we can default enabled to True.